### ISSUE TITLE
show-language plugin does not work with Coy theme

### ISSUE THREAD

**Issue by uranusjr** [PrismJS/prism#667]
I think this is because the Coy theme uses `:before` to display a slanted background decoration, which the show-language plugin uses to display the language label.

**Comment by nauzilus** [comment]
You're right :( I'll look at fixing this soon. What I'm thinking of doing will mean only the right hand side will have the shadow, so only the bottom right corner will appear to be lifting from the page when using show-language with coy theme. But that's probably an ok compromise.

Actually another PR I'm waiting on #650 would make this easier, since it allows plugins to have theme specific CSS, so I could target the fix just for coy theme :)

**Comment by nauzilus** [comment]
Was hoping to get #650 merged first then I'd fix this in the theme specific CSS, but #703 has been merged so this can be closed I should think.

+@uranusjr

**Comment by uranusjr** [comment]
Just tried the latest version via prismjs.com and this is indeed fixed. I’ll close this.

BTW I noticed another problem with the Coy theme on the website. It overflows the content section and overlaps the theme chooser on the right:

![screen shot 2015-08-21 at 12 17 05](https://cloud.githubusercontent.com/assets/605277/9401215/8fea371c-47fe-11e5-8b84-49d7d098c278.png)

For comparison, this is how the default theme looks:

![screen shot 2015-08-21 at 12 17 44](https://cloud.githubusercontent.com/assets/605277/9401227/b24cb74e-47fe-11e5-8810-4f2aafb62598.png)

**Comment by Golmote** [comment]
@nauzilus, sorry for the merge order. I usually don't merge new components or new plugins without @LeaVerou's approval.
